### PR TITLE
57 Edit Create Parent

### LINKED ISSUE #57: EC.4 Create Edit/Create Parent
## User Story
As a user, I want to visit the edit/create page and have everything laid out, so I can navigate create and edit my study set.

##Sub-Tasks
- [ ] Create Edit/Create Flashcard Page
- [ ] Add NavBar

## Notes
- Depends on #12

### PR DESCRIPTION
**Pull request by JeremyLin003**
Fixes #57 

- Created Edit Create parent component
- Placed visual placeholders to emulate final design
- Added minimal routing to navigate to Edit Create page from NavBar
- I will be updating the visuals and functionality of this page further in #20  
 
image

### CODE REVIEW

File: src/app/edit-create-study-set/edit-create-study-set.component.html [hunk 1]
```diff
@@ -0,0 +1,59 @@
+<head>
+  <title>Edit Create Study Set</title>
+</head>
+
+
+<body>
+<div class = "body-div">
+  <div class = "upper-components">
+    <div class ="first-row">
+      <p class = "page-title">Create Study Set</p>
```
Comment (ashkjones) [inline src/app/edit-create-study-set/edit-create-study-set.component.html:10]: change `p` to `h1` and set a global style in `src/app/styles.scss` for all headings `h1`.

File: src/app/edit-create-study-set/edit-create-study-set.component.scss [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+  .first-row{
+    display: flex;
+    align-items: center;
+    vertical-align: baseline;
+  }
+
+  .page-title{
+    font: mat.get-theme-typography($theme, headline-large);
+    line-height:0em;
+  }
```
Comment (ashkjones) [inline src/app/edit-create-study-set/edit-create-study-set.component.scss:22]: make this the global style for `h1` in the `src/styles.scss` file. In `styles.scss` add:
```css
h1 {
  font: mat.get-theme-typography(theme.$ponder-light-theme, headline-large);
  line-height:0em;
}
```

**Review by ashkjones — CHANGES_REQUESTED** [review]
I requesting some code be added to components. When you implement the tabs on your next PR, please make sure to make them a component so they can be reused on the view pages as well.

**Review by B-Yamawaki — APPROVED** [review]
Good Job

**Pull request merged**